src/config/index.ts

```diff
@@ -239,7 +239,11 @@ export const CONTACT_EMAIL = "#contact_email";
 export const CREATED_DATE = "#createdDate-createdDateDay";
 export const INCORPORATION_COUNTRY = "#incorporation_country";
 export const ENTITY_NAME = "#entity_name";
-export const ROLE_WITHIN_TRUST = "#role_within_trust";
```

Comment:
Is there a reason for this change, the former assignment seems more descriptive

Reply:
This var wasn't being used anywhere currently and the value for `role_with_trust `on the web page is still `type` in the html so the change link needs to look for type. 
I agree with you though and didn't really like the change to much myself but thought it best for another story where we can maybe refactor the entire use of `type` for `role_within_trust`.

What do you think?

Reply:
Fair point. We probably want to send Sarah these refactorings in groups for stories.